### ISSUE TITLE
Support third-party plugins

### ISSUE THREAD

**Issue by swissspidy** [wearerequired/digest#1]
Plugins that we could support, off the top of my head:
- [ ] Antispam Bee
- [ ] Antivirus
- [ ] bbPress
- [ ] BuddyPress
- [ ] Checksum Verifier
- [ ] Easy Digital Downloads
- [ ] Gravity Forms
- [ ] Jetpack (Security, Forms, etc.)
- [ ] Stream
- [ ] User Feedback
- [ ] WooCommerce

**Comment by neverything** [comment]
This would be ideal and as you mentioned pluginkollektiv/antivirus#5 the combination of AntiVirus and Checksum Verifier, could be very powerful.

**Comment by swissspidy** [comment]
Yeah, the goal is to be create the go-to solution for digest notifications in WordPress.

We could also support bbPress and BuddyPress (see also https://github.com/dimadin/bbpress-digest).

**Comment by swissspidy** [comment]
Add Antispam Bee to the list, see #8

**Comment by swissspidy** [comment]
Of course we could support our very own [User Feedback](https://github.com/wearerequired/user-feedback) plugin as well 😄
